test/infrastructure/docker/hack/verify-spelling.sh

```diff
@@ -38,7 +38,7 @@ URL="https://github.com/client9/misspell"
 echo "Cloning ${URL} in ${TMP_DIR}..."
```

Comment:
This script can moved to hack/ in capi repo root.

Reply:
This shouldn't be needed as a script, we should enable it in golangci-lint https://github.com/golangci/golangci-lint#disabled-by-default-linters--e--enable

Reply:
Same probably applies to `gofmt`, `goimports, `govet`, and so on

Reply:
Agree. We've enabled these linters. https://github.com/kubernetes-sigs/cluster-api/blob/master/.golangci.yml
We can delete these scripts.